packages/cli/lib/validators/logValidatorResults.js

```diff
@@ -0,0 +1,51 @@
+const { logger } = require('@hubspot/cli-lib/logger');
+const { VALIDATION_RESULT } = require('./constants');
+
+function logResultsAsJson(results) {
+  let success = true;
+  const resultObj = results.reduce((acc, result) => {
+    if (!acc[result.validator]) {
+      if (success && result.result !== VALIDATION_RESULT.SUCCESS) {
+        success = false;
+      }
+      acc[result.validatorKey] = [];
+    }
+    const {
+      validatorName: __omit1, // eslint-disable-line no-unused-vars
```

Comment:
again, sorry for my lack of javascript knowledge here, but what is `__omit1` and `__omit2`

Reply:
So this is a way to remove fields from an object. It's basically doing the same thing as [_.omit()](https://underscorejs.org/#omit). I'm destructuring off the fields that I want to omit, then using `...rest` to get the resulting object w/out those fields. `__omit1` and `__omit2` are just names I used to try to make it more clear that these variables aren't being used anywhere.

Reply:
cool! thanks for that explanation! Makes sense, we can throw away the ValidatorKey since that's what we're consolidating everything on and we don't need to know the ValidatorName anymore at this point